utils/docker/README.md

```diff
@@ -25,6 +25,9 @@ see https://github.com/pmem/ndctl/tags.
 For testing Threading Building Blocks, TBB library version tag must be passed as parameter,
 see https://github.com/01org/tbb/tags.
 
+* **HOG_MEMORY** - Controls behavior of memkind with regards to returning memory to underlying OS. Setting **HOG_MEMORY** to 1 causes
```

Comment:
It will *never* release the memory? What happens when you destroy the kind?

What's the difference between this variable and memory usage policy?